spec/workflow/choice_rule_spec.rb

```diff
@@ -225,30 +232,40 @@
       end
 
       context "with IsNumeric" do
-        let(:choices) { [{"Variable" => "$.foo", "IsNumeric" => true, "Next" => "FirstMatchState"}] }
+        let(:positive) { true }
```

Comment:
This being named `positive` is confusing later on when we say `positive: false`.  Trying to think of better names, but can't think of one at the moment.  I understand the sentiment you're going after, but because of the construction of the tests where this is somewhat out of context, it just feels very confusing

Reply:
My first thought was "predicate" as in https://en.wikipedia.org/wiki/Predicate_(mathematical_logic)

Reply:
LOL - or how about just "boolean"